croud/regions/commands.py

```diff
@@ -138,7 +139,7 @@ def regions_delete(args: Namespace) -> None:
         print_response(
             data=data,
             errors={
-                "message": region_errors.get("message"),
+                "message": cast(str, region_errors.get("message")),
```

Comment:
That might be a viable alternative instead of using a `cast`. wdyt?
```suggestion
                "message": region_errors.get("message", "unknown error"),
```

Reply:
Yes, it makes sense to have a meaningful message.

Reply:
Hi. I changed the code according to the suggestion. The patch is now ready for review, if you don't have any other objections.

Reply:
> Bumping the mypy version should be done in `.github/dependabot.yml` too.

Right. Done.